frontend/public/components/resource-quota.jsx

```diff
@@ -12,12 +14,31 @@ const { common } = Kebab.factory;
 const menuActions = [...common];
 
 const quotaKind = quota => quota.metadata.namespace ? referenceForModel(ResourceQuotaModel) : referenceForModel(ClusterResourceQuotaModel);
+const gaugeChartThresholds = {warn: 90,error: 100};
```

Comment:
Does this only show an error when over 100? (Exactly 100 should probably not be an error IMO.)

style nit:

```suggestion
const gaugeChartThresholds = {warn: 90, error: 100};
```

Reply:
Also have you confirmed that the gauge looks ok if you go over 100%?

Reply:
Hi @spadgett, great catch!  Pls see last commit, had to adjust to warn at 100%, error when > 100%. -thanks

![image](https://user-images.githubusercontent.com/12733153/49809488-068c1700-fd2d-11e8-9c66-16d55604a48e.png)

Reply:
arrg...was on wrong branch...hang on

Reply:
> had to adjust to warn at 100%, error when > 100%. -thanks

Looks great 👍